xml/issue4383.xml

```diff
@@ -142,7 +142,123 @@ struct <i>cw-operators</i> {
 </ol>
 </superseded>
 
-<note>2025-11-04; Zach provides improved wording</note>
+<p>
```

Comment:
lines 145-261 should be moved inside the `<resolution>` element, instead of within the `<discussion>`.

The old resolution that I moved to the `<superseded>` and the `<note>` belong in the `<discussion>`, but there's a special element for the current P/R.